code/controllers/subsystem/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -574,7 +574,7 @@ GLOBAL_VAR_INIT(revolutionary_win, FALSE)
 /// Checks for revhead loss conditions and other antag datums.
 /datum/dynamic_ruleset/roundstart/revs/proc/check_eligible(datum/mind/M)
 	var/turf/T = get_turf(M.current)
-	if(!considered_afk(M) && considered_alive(M) && is_station_level(T.z) && !M.antag_datums?.len && !HAS_TRAIT(M, TRAIT_MINDSHIELD))
+	if(!considered_afk(M) && considered_alive(M) && is_station_level(T.z) && !M.antag_datums?.len && !HAS_TRAIT(M, TRAIT_UNCONVERTABLE))
```

Comment:
All instances of `HAS_TRAIT(x, TRAIT_UNCONVERTABLE)` should be `HAS_MIND_TRAIT(x, TRAIT_UNCONVERTABLE)`

Reply:
Good call, I didn't realize that `HAS_MIND_TRAIT` checked both the mind and the mob itself